Remove Makefile reference to third-parties that are removed for Debian package
--- a/third-party/Makefile
+++ b/third-party/Makefile
@@ -15,43 +15,43 @@
 
 default: all
 
-all: test-venv chpldoc-venv gasnet gmp hwloc jemalloc qthread llvm
+all: test-venv chpldoc-venv hwloc jemalloc qthread
 
 clean: FORCE
 	cd chpl-venv && $(MAKE) clean
-	cd gasnet && $(MAKE) clean
-	cd gmp && $(MAKE) clean
+	#cd gasnet && $(MAKE) clean
+	#cd gmp && $(MAKE) clean
 	cd hwloc && $(MAKE) clean
 	cd jemalloc && $(MAKE) clean
-	cd llvm && $(MAKE) clean
-	cd massivethreads && $(MAKE) clean
+	#cd llvm && $(MAKE) clean
+	#cd massivethreads && $(MAKE) clean
 	cd qthread && $(MAKE) clean
 	cd re2 && $(MAKE) clean
-	cd fltk && $(MAKE) clean
+	#cd fltk && $(MAKE) clean
 
 cleanall: FORCE
 	cd chpl-venv && $(MAKE) cleanall
-	cd gasnet && $(MAKE) cleanall
-	cd gmp && $(MAKE) cleanall
+	#cd gasnet && $(MAKE) cleanall
+	#cd gmp && $(MAKE) cleanall
 	cd hwloc && $(MAKE) cleanall
 	cd jemalloc && $(MAKE) cleanall
-	cd llvm && $(MAKE) cleanall
-	cd massivethreads && $(MAKE) cleanall
+	#cd llvm && $(MAKE) cleanall
+	#cd massivethreads && $(MAKE) cleanall
 	cd qthread && $(MAKE) cleanall
 	cd re2 && $(MAKE) cleanall
-	cd fltk && $(MAKE) cleanall
+	#cd fltk && $(MAKE) cleanall
 
 clobber: FORCE
 	cd chpl-venv && $(MAKE) clobber
-	cd gasnet && $(MAKE) clobber
-	cd gmp && $(MAKE) clobber
+	#cd gasnet && $(MAKE) clobber
+	#cd gmp && $(MAKE) clobber
 	cd hwloc && $(MAKE) clobber
 	cd jemalloc && $(MAKE) clobber
-	cd llvm && $(MAKE) clobber
-	cd massivethreads && $(MAKE) clobber
+	#cd llvm && $(MAKE) clobber
+	#cd massivethreads && $(MAKE) clobber
 	cd qthread && $(MAKE) clobber
 	cd re2 && $(MAKE) clobber
-	cd fltk && $(MAKE) clobber
+	#cd fltk && $(MAKE) clobber
 
 depend:
 
